app/src/utils/LinkWithStickyParams.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import useSearchParams from "./useSearchParams";
+import { Link } from "react-router-dom";
+
+export const STICKY_VERSION_PARAM = "sv";
+
+function LinkWithStickyParams(props: {
+  to: string;
+  children: React.ReactNode;
+}) {
+  let params = useSearchParams();
```

Comment:
This function is only two lines long- is there a strong benefit to having it in its own separate file?

Reply:
Moved both into urlutils file